salt/modules/virt.py

```diff
@@ -1200,6 +1197,8 @@ def _get_merged_nics(hypervisor, profile, interfaces=None, dmac=None):
 def init(name,
          cpu,
          mem,
+         os_type=None,
```

Comment:
These new parameters need to go at the end of the list so we don't break things for people if they are using positional arguments.